proposals/lip-0043.md

```diff
@@ -112,8 +112,9 @@ The `ownName` property sets the name of the sidechain in its own state according
 | Name                                  | Type   | Value                | Description                                        |
```

Comment:
I disagree, I think it is better as it is now, since the rationale is much more readable and gives a complete overview. Then the specs are the "details".

Reply:
I would not change the order now. I added the decription of `extraCommandFee`